packages/@okta/vuepress-site/docs/api/resources/policy/index.md

```diff
@@ -52,7 +52,7 @@ curl -v -X GET \
 
 
 HTTP 200:
-<a href="#PolicyObject">Policy Object</a>
+<a href="#policy-object">Policy Object</a>
```

Comment:
Can we just use the markdown link style here, like in the rest of the doc?

Reply:
Applied following to replace anchors

```
perl -pi -e "s/<a\s+href=\"([^\"]+)\">([^<]+)<\/a>/[\2](\1)/g" packages/@okta/vuepress-site/docs/api/resources/policy/index.md
```